doc/api/child_process.md

```diff
@@ -612,6 +612,10 @@ pipes between the parent and child. The value is one of the following:
    will enable [`process.send()`][], [`process.disconnect()`][],
    [`process.on('disconnect')`][], and [`process.on('message')`] within the
    child.
+
+   *Note*: Accessing the IPC channel fd in any way other than
+   [`process.send()`][] or using the IPC channel with a child process
+   that is not a Node.js instance is undefined.
```

Comment:
What does "undefined" mean? Did you mean "undefined behavior"?

Reply:
Nit: I'd prefer we remove `*Note*:` and just let the paragraph/sentence stand on its own. There are already 29 Notes in the doc as it is. We overuse that pattern. And the italicized `Note:` doesn't really call attention to things IMO.

Reply:
Using the IPC channel with a non-node child process used to be documented, and basically still is, though it looks like the purpose of the text got lost a bit in various rewrites:

> using the environment variable NODE_CHANNEL_FD on the child process. The input and output on this fd is expected to be line delimited JSON objects

It now sounds like the env var and the JSON protocol is only to be used by ***node.js*** child processes, but that is a bit odd, if so, this information would be entirely implementation detail (like the protocol used internally by the cluster module, which is entirely undocumented). The only purpose to describe the env var and protocol so specifically is so that it *can* be used by non-node.js child processes. If node exposes when the protocol is abused, that's a bug - though a bug of the "doctor it hurts when I hit myself with a hammer" variety - should be fixed, but not very severe.

Reply:
I would prefer if we stated explicitly that the IPC channel should not be used if not through process.send()

Reply:
minor suggestion: I would just drop the `*Note*: ` prefix.